react/react_testing/mocking_callbacks_and_components.md

```diff
@@ -16,81 +16,58 @@ If you've been following along with our lessons so far, the concept of mocking h
 
 #### Testing Callback Handlers
 
-Callbacks are ubiquitous. Every avenue of user interaction involves callbacks. Sometimes they're passed in as props to alter state of the parent component. Consider this simple input component:
+Callbacks are ubiquitous. Every avenue of user interaction involves callbacks. Sometimes they're passed in as props to alter state of the parent component. Consider this simple button component:
 
 ~~~jsx
-// FavoriteInput.jsx
-
-import React from "react";
-
-const FavoriteInput = ({ onChange: onInputChange, id }) => {
-  const inputHandler = (event) => onInputChange(event.target.value);
+// CustomButton.jsx
 
+const CustomButton = ({ onClick, text}) => {
   return (
-    <label htmlFor={id}>
-      What is your favorite wild animal?
-      <input id={id} onChange={inputHandler} />
-    </label>
+    <button onClick={onClick}>{text}</button> 
```

Comment:
Nit: I would have the children here be passed as children rather than a `text` prop. Or maybe just have the button text be hardcoded for this and just pass the `onClick` to it for maximum simplicity in the examples.

Reply:
Done!